test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -979,10 +1028,6 @@ def setup_rnn_tests():
                 ('lstm', 'lstm', {}),
                 ('gru', 'gru', {})
         ):
-            # This is a hack to skip elman_rnn bidirectional tests for now
-            # TODO: Revert this once elman_rnn bidirectional issue is fixed
-            if base == 'elman' and bidirectional[1] == 'bidirectional':
```

Comment:
Why do we combine two changes together?

Reply:
1. for enabling disabled tests, it is because clamp tests needs ort version bump, and with new versions some disabled tests can be enabled now.
2. for enabling flaky lstm/gru/rnn tests, it's because the fix for relaxing tolerance is merged.

Both changes are small so I included them here, I'm ok with removing them and submit a separate PR.

Reply:
Yes, please submit a separate PR for enabling tests, thanks.

Reply:
removed test changes from this one.